### PR TITLE
Proposed move of deidservice under healthcareapis

### PR DESCRIPTION
**Pull request by chrisribe**
This is a proposed move of deidservice extension under healthcareapis

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by chrisribe** [comment]
Related AAZ PR:
https://github.com/Azure/aaz/pull/620

File: src/healthcareapis/azext_healthcareapis/azext_metadata.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-    "azext.minCliCoreVersion": "2.15.0"
+    "azext.minCliCoreVersion": "2.61.0"
```
Comment (kairu-ms) [inline src/healthcareapis/azext_healthcareapis/azext_metadata.json:2]: The identity should be used in version above 2.66.0
```suggestion
    "azext.minCliCoreVersion": "2.66.0"
```

**Comment by kairu-ms** [comment]
Please remove the extension in `https://github.com/Azure/azure-cli-extensions/blob/main/src/index.json` file as well

File: src/healthcareapis/azext_healthcareapis/azext_metadata.json (hunk 1 above)
Comment (chrisribe) [reply src/healthcareapis/azext_healthcareapis/azext_metadata.json:2]: Done

**Comment by chrisribe** [comment]
> Please remove the extension in `https://github.com/Azure/azure-cli-extensions/blob/main/src/index.json` file as well

Oh nice catch ! Removed

**Comment by chrisribe** [comment]
azure-cli 2.66.0 is release I think we can release this PR now right ?
https://github.com/Azure/azure-cli/releases/tag/azure-cli-2.66.0
@kairu-ms, @yonzhan, @wangzelin007, @jsntcy, @zhoxing-ms,

File: src/deidservice/azext_deidservice/azext_metadata.json [hunk 2]
```diff
@@ -1,4 +0,0 @@
-{
-    "azext.isPreview": true,
-    "azext.minCliCoreVersion": "2.61.0"
-}
```
Comment (kairu-ms) [inline src/deidservice/azext_deidservice/azext_metadata.json:4]: Please keep this file and chnage the "azext.minCliCoreVersion to "2.66.0"

**Review by kairu-ms — APPROVED** [review]

**Comment by chrisribe** [comment]
> Ok well my index.json revert has caused issues I think I need to merge main again. Working on that now....
Ok fixed and rebased on main

**Review by jovinson-ms — APPROVED** [review]

**Pull request merged**